addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -420,7 +427,8 @@ def _get_accrual_plan_level_work_entry_prorata(self, level, start_period, start_
         else:
             planned_worked = worked
         left = self.employee_id.sudo()._get_leave_days_data_batch(start_dt, end_dt,
```

Comment:
When you check the last version (with which you have a conflict here) it seems it as already been fixed using the ``_get_calendar`` method actually.

```py
        left = self.employee_id.sudo()._get_leave_days_data_batch(start_dt, end_dt, calendar=self.employee_id._get_calendar(start_dt),
            domain=[('time_type', '=', 'leave')])[self.employee_id.id]['hours']
```